### PR TITLE
changelog: update for v0.34.4

### PR DESCRIPTION
**Pull request by tessr**
Updates the changelog in preparation for v0.34.4.

### CODE REVIEW

**Review by cmwaters — APPROVED** [review]

**Comment by cmwaters** [comment]
Should the links be issues or PR's?

**Comment by tessr** [comment]
> Should the links be issues or PR's?

They are the same thing - I have a preference for pointing to PRs, though

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,30 @@
 # Changelog
 
-## v0.34.3 
+## v0.34.4
+
+*February 11, 2021*
+
```
Comment (melekes) [inline CHANGELOG.md:6]: Should we say that this release fixes a memory leak and all clients are recommended to upgrade?

Comment (tessr) [reply CHANGELOG.md:6]: Mmm yes, I like this! Will update (in another PR unfortunately).

**Pull request merged**